docs/README.md

```diff
@@ -44,6 +44,26 @@ If the files in `pkg/apis` are updated we need to run `codegen` scripts
 ```shell script
     make TARGET=openshift apply
 ```
+### Install Tekton components
+Operator provides an option to choose which components needs to be installed by specifying `profile`.
+
+`profile` is an optional field and supported `profile` are
+* **basic** 
+* **default**
+* **all**
+
+1. If profile is `basic` **TektonPipeline** will be installed
+1. If profile is `default` or `" "` **TektonPipeline** and **TektonTrigger** will be installed
+1. If profile is `all` then all the Tekton Components installed
```

Comment:
Will the profiles be a bit different for the kubernetes platform vs openshift platform ? 
aka What would be the profile for what is done by OpenShift Pipelines as of today ?

Reply:
extension magic. :tada: 
for example
`all` profile installs addons on openshift (no dashboard), but on k8s it installs dashboard but no addons.

the parts which are not common are handled as extensions.
https://github.com/tektoncd/operator/pull/179/files#diff-a100309e66fd0aa1063e2e110fd0d6d97eaa6516bf10a9f61285828a7a0697c1
and
https://github.com/tektoncd/operator/pull/179/files#diff-9180de7e95281f01fe01e488cdcc0c7119c98362aa6a33308095057d2cc6918c

Reply:
profile is common for both the openshift and kubernetes 

If `profile` is **all** 
On Openshift -> Pipeline, Triggers and Addons will be installed
On Kubernetes -> Pipeline, Triggers and Dashboard will be installed